enclave/crypto/mbedtls/cert.c

```diff
@@ -414,91 +414,76 @@ typedef bool (*parse_extensions_callback_t)(
 
 typedef struct _find_extension_args
 {
-    oe_result_t result;
     const char* oid;
     uint8_t* data;
-    size_t* size;
-} FindExtensionArgs;
+    size_t size;
+} find_extension_args;
```

Comment:
```suggestion
} find_extension_args_t;
```
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md#naming-conventions says in point 7:

> always create a typedef with the suffix `_t`